docs/apis/streaming/connectors/kafka.md

```diff
@@ -142,18 +142,24 @@ for querying the list of topics and partitions.
 For this to work, the consumer needs to be able to access the consumers from the machine submitting the job to the Flink cluster.
 If you experience any issues with the Kafka consumer on the client side, the client log might contain information about failed requests, etc.
 
-##### The `DeserializationSchema`
+##### **The `DeserializationSchema`**
```

Comment:
`kafka.md` uses very low level headings. The batch and streaming guides use level 2 headings for top-level headings while `kafka.md` uses level 4. I think the solution is to reduce heading level by 2 on all levels in this document, which would yield
```
### The `DeserializationSchema`
```